docs/website/docs/dlt-ecosystem/destinations/athena.md

```diff
@@ -0,0 +1,19 @@
+---
+title: AWS Athena / Glue Catalog
+description: AWS Athena `dlt` destination
+keywords: [aws, athena, glue catalog]
+---
+
+# AWS Athena
+
+### Setup:
```

Comment:
As a minimum infra the user need to have the following:
* An S3 bucket to store target data written by dlt (via athena)
* An S3 bucket used by athena to store query results

Some optional configurations
* Glue target database - by default athena uses `default` database
* Athena workgroup - by default the primary workgroup can be used